lacquer/tree/visitor.py

```diff
@@ -516,7 +517,12 @@ def visit_query_specification(self, node, context):
         if node.where:
             self.process(node.where, context)
         if node.group_by:
-            for grouping_element in node.group_by.grouping_elements:
+            grouping_elements = []
+            if isinstance(node.group_by, SimpleGroupBy):
+                grouping_elements = node.group_by.columns
+            elif isinstance(node.group_by, GroupingSets):
```

Comment:
I don't (yet) have GroupingSets in the parser, but we can leave this in for when I get there.